### PR TITLE
NAS-111266 / 21.08 / Add 2 dropdowns

### PR DESCRIPTION
**Pull request by RehanY147**
NAS-111266

Test adding the new fields to network interface
Also test updating the fields once interface has been created

### CODE REVIEW

**Review by AlexKarpov98 — COMMENTED** [review]
@RehanY147 It does not seem to save the new options to middleware correctly.
When you edit the interface having these 2 options, the saved values are not shown on these select controls.

File: src/app/pages/network/forms/interfaces-form.component.ts [hunk 1]
```diff
@@ -485,6 +520,22 @@ export class InterfacesFormComponent extends ViewControllerComponent implements
       entityForm.setDisabled('name', true);
       entityForm.setDisabled('type', true, true);
     }
+
+    this.entityForm.formGroup.get('lag_protocol').valueChanges.pipe(untilDestroyed(this)).subscribe((value: string) => {
```
[the remark this answers is not in the document]
Comment (AlexKarpov98) [reply src/app/pages/network/forms/interfaces-form.component.ts:524]: @RehanY147 This issue was not fixed yet.
add_interface

**Review by AlexKarpov98 — CHANGES_REQUESTED** [review]

**Comment by RehanY147** [comment]
@AlexKarpov98 Nice testing. Fixed the issue. Ready for review.

**Review by AlexKarpov98 — APPROVED** [review]
@RehanY147 LGTM

**Pull request merged**